package.json

```diff
@@ -9,7 +9,7 @@
     "build:common": "shx rm -rf dest && npm run build:env && npm run fetch && run-p build:images build:pug build:sass && npm run build:healthcheck && npm run build:contribute",
     "build:contribute": "shx cp contribute.json dest/",
     "build:healthcheck": "node scripts/build-healthcheck.js > dest/healthcheck.html",
-    "build:images": "shx rm -rf dest/_images && shx cp -r source/images dest/_images",
+    "build:images": "shx rm -rf dest/_images && shx mkdir -p dest/ && shx cp -r source/images dest/_images",
```

Comment:
Revisiting this I can see there's a slightly DRY-er way. Can you try this?

https://github.com/mozilla/network/pull/542